kits/arm/ex_impedance_control.py

```diff
@@ -54,11 +51,26 @@
 if enable_logging:
     arm.group.start_log('dir', 'logs', mkdirs=True)
 
-print('Commanded gravity-compensated zero force to the arm.')
-print('  b2 - Toggles an impedance controller on/off:')
-print('          ON  - Apply controller based on current position')
-print('          OFF - Go back to gravity-compensated mode')
-print('  b1 - Exits the demo.')
+# Print Instructions
+instructions = """Commanded gravity-compensated zero force to the arm.
+B1 - Exit Demo
+B2 - Toggles an impedance controller on/off:
+          ON  - Apply controller based on current position
+          OFF - Go back to gravity-compensated mode
+"""
+print(instructions)
+
+m.set_button_label(1, 'Exit')
+m.set_button_label(2, 'Toggle Impedance')
+
+for button in range(3, 9):
+    m.set_button_label(button, '')
+
+for axis in range(1, 9):
+    m.set_axis_label(axis, '')
+  
+m.set_button_mode(2, 1)
```

Comment:
One other thing -- the previous calls to `set_button_mode` were clearer to me, where the type of button mode was explicitly set, instead of a magic number.